server/graphql/v1/mutations.js

```diff
@@ -193,8 +194,9 @@ const mutations = {
         description: 'The new email address for user',
       },
     },
-    resolve: (_, { email }, { remoteUser }) => {
-      return updateUserEmail(remoteUser, email);
+    resolve: async (_, { email }, req) => {
+      await twoFactorAuthLib.validateRequest(req);
```

Comment:
The check for `req.remoteUser` in `validateRequest` I suggested above seems important to have a clean error message for this one